platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -2047,9 +2047,19 @@
             var exists;
 
             if ($scope.advancedSettings !== null && $scope.lookup !== undefined && $scope.lookup.lookupName !== undefined) {
```

Comment:
@sebbrudzinski @MichalKus I believe that we can get rid of it here. If so, I think the timeout should be removed as well and I would left just the validation of the lookup name